operations/helm/charts/mimir-distributed/templates/nginx/nginx-svc.yaml

```diff
@@ -23,6 +23,9 @@ spec:
   {{- if and (eq "LoadBalancer" .Values.nginx.service.type) .Values.nginx.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.nginx.service.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.nginx.service.trafficDistribution }}
+  trafficDistribution: {{ .Values.nginx.service.trafficDistribution }}
```

Comment:
this is a deprecated component, can you not modify it? it will incentivize people to migrate to the new gateway deployment. We're also removing it in 3.0, so you might end up conflicting with `main`